.github/workflows/reviewdog.yml

```
@@ -4,6 +4,7 @@ env:
   DISABLE_SPRING: 1
 jobs:
   reviewdog:
+    if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)